src/treelearner/kernels/histogram_16_64_256.cu

```diff
@@ -0,0 +1,1059 @@
+/*!
+ * ibmGBT: IBM CUDA Accelerated LightGBM
```

Comment:
This is now becomming a part of official LightGBM, please remove this name to avoid confusion.